### ISSUE TITLE
Duplicate import statement for `google.longrunning.operations_pb2` in `python-logging`

### ISSUE THREAD

**Issue by parthea** [googleapis/gapic-generator-python#1727]
In PR https://github.com/googleapis/python-logging/pull/746, there is a duplicate import statement for `google.longrunning.operations_pb2`. See the review comment [here](https://github.com/googleapis/python-logging/pull/746#discussion_r1190481195).

**Comment by parthea** [comment]
I was able to re-create the issue in https://github.com/googleapis/gapic-generator-python/pull/1726/commits/5e03e71cdcd0db23592c30a93c76704eb1c0c1ce using the latest version of `googleapis/googleapis` . #1726 fixes the problem. I had to back out the update of `googleapis/googleapis` in #1726 because it caused a large change to the golden files. The update of `googleapis/googleapis` should be done in a separate PR.
